srcpkgs/apache-jmeter/template

```diff
@@ -0,0 +1,42 @@
+# Template file for 'apache-jmeter'
+pkgname=apache-jmeter
+version=5.1.1
+revision=1
+hostmakedepends="openjdk apache-ant"
+depends="virtual?java-runtime"
+short_desc="Application to load test functional behavior and measure performance"
+maintainer="Kyle Nusbaum <knusbaum+void@sdf.org>"
+license="Apache-2.0"
+homepage="https://jmeter.apache.org/"
+distfiles="http://apache.osuosl.org//jmeter/source/apache-jmeter-${version}_src.tgz"
+checksum=133fee3ccc300c759b5fbba4dd870db8e69aa0f72eca080fb37a8fb56c6caf6b
+
+do_configure() {
+	ant download_jars
+}
+
+do_build() {
+	ant install
+}
+
+do_install() {
+	rm -Rf src test build
+
+	vlicense LICENSE
+	vmkdir /opt/apache-jmeter
+	vcopy . /opt/apache-jmeter
+
+	vmkdir /usr/bin
+	ln -s /opt/apache-jmeter/bin/jmeter ${DESTDIR}/usr/bin/jmeter
+}
+
+apache-jmeter-doc_package() {
```

Comment:
Is there sufficient documentation to justify the split?  Normally we'd keep the docs with the package unless they're huge.

Reply:
They're 57MB unpacked. It seemed sizeable to me, but I'm happy to leave them in the main package if you want. Please let me know.

Reply:
Hmm, that is sizeable, go ahead and leave it split.